src/components/Chart/styles/RoundsChart.module.scss

```diff
@@ -0,0 +1,35 @@
+.container {
+  margin-bottom: 30px;
+}
+
+.axis {
+  path, line {
+    fill: none;
+    stroke: #dee2e6; // Match table border style
+    shape-rendering: crispEdges;
+  }
+
+  .domain {
```

Comment:
Is this class used? Can’t see it anywhere.

Reply:
I'll remove.